msteams-platform/teams-faq.md

```diff
@@ -587,6 +587,23 @@ For more information about Node js code sample, see [Bot SSO quick-start](https:
 Upgrade the Teams JavaScript client SDK to (@microsoft/teams-js": "^2.0.0") to resolve the issue.
 <br>
 </details>
+<br>
+<details> 
+<summary>Which tab gets pinned if both configurable tab and personal (static) tab are defined in the app manifest for a specific scope?</summary>
+
+<br>
+If you have both configurable tab and personal (static) tab defined in your app manifest for a specific scope, Teams pins the personal (static) tab by default.
```

Comment:
As mentioned above, do a global replace anywhere you use "personal (static)" with just "static".

Reply:
Updated